streams/src/test/java/org/apache/kafka/streams/state/internals/NamedCacheTest.java

```diff
@@ -45,7 +44,7 @@
 
     @Before
     public void setUp() throws Exception {
-        streamMetrics = new MockStreamsMetrics(new Metrics());
+        streamMetrics = new MockStreamsMetrics();
```

Comment:
Similarly as in `StreamTaskTest`, we should just use a different `cache` object in the `testMetrics` function than the class field `cache` which gets a real `StreamsMetricsImpl` directly.